app/components/status_tags/base_component.rb

```diff
@@ -40,7 +41,7 @@ def colour_class
     end
 
     def task_list?
-      true
+      task_list != false
```

Comment:
I think this might return true if task_list is nil, which probably isn't what we want? Maybe better to check if it's true:

```suggestion
      task_list == true
```

Reply:
If it's nil or true we want this css class `app-task-list__task-tag`.
But I agree this looks a bit confusing so i'll try to rework this